packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -70,21 +79,36 @@ class DropdownItem extends UIComponent<ReactProps<DropdownItemProps>, any> {
     return (
       <ListItem
         className={classes.root}
-        header={header}
+        header={Box.create(header, {
+          defaultProps: {
+            className: DropdownItem.slotClassNames.header,
```

Comment:
Can we apply here also: `styles.header`? That way people will be able to define a slot in the styles for the DropdownItem if they need some overrides. The same applies for the `media` and `content`